src/Aspire.Hosting.Garnet/Aspire.Hosting.Garnet.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <PackageTags>aspire hosting garnet</PackageTags>
+    <Description>Garbet® support for .NET Aspire.</Description>
```

Comment:
Can you add a README for the library as well? see https://github.com/dotnet/aspire/pull/3828

Reply:
Yes, I'll try to add a README for the library

Reply:
I look forward to your comments regarding the README